### ISSUE TITLE
problems with tests

### ISSUE THREAD

**Issue by vwkoppejan** [usnistgov/fipy#515]
Test are not finishing:

> python -c "import fipy; fipy.test()"
running egg_info
creating /tmp/tmpLdTKTE/FiPy.egg-info
writing /tmp/tmpLdTKTE/FiPy.egg-info/PKG-INFO
writing top-level names to /tmp/tmpLdTKTE/FiPy.egg-info/top_level.txt
writing dependency_links to /tmp/tmpLdTKTE/FiPy.egg-info/dependency_links.txt
writing manifest file '/tmp/tmpLdTKTE/FiPy.egg-info/SOURCES.txt'
reading manifest file '/tmp/tmpLdTKTE/FiPy.egg-info/SOURCES.txt'
writing manifest file '/tmp/tmpLdTKTE/FiPy.egg-info/SOURCES.txt'
running test
running build_ext
fipy version 3.1.3
numpy version 1.11.3
pysparse version 1.3-dev
scipy version 0.19.0
matplotlib version 2.0.2
gist is not installed
mpi4py version 2.0.0
Trilinos version: 12.10.1
PyTrilinos version: 12.8
mayavi version 4.5.0
gmsh version 2.15.0
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/data/localhome/vwkoppejan/software/python/anaconda2/envs/fipy2-env/lib/python2.7/site-packages/fipy/__init__.py", line 164, in test
    cmdclass={'test': _TestClass(_test)})
  File "/data/localhome/vwkoppejan/software/python/anaconda2/envs/fipy2-env/lib/python2.7/distutils/core.py", line 151, in setup
    dist.run_commands()
  File "/data/localhome/vwkoppejan/software/python/anaconda2/envs/fipy2-env/lib/python2.7/distutils/dist.py", line 953, in run_commands
    self.run_command(cmd)
  File "/data/localhome/vwkoppejan/software/python/anaconda2/envs/fipy2-env/lib/python2.7/distutils/dist.py", line 972, in run_command
    cmd_obj.run()
  File "/data/localhome/vwkoppejan/software/python/anaconda2/envs/fipy2-env/lib/python2.7/site-packages/setuptools/command/test.py", line 211, in run
    self.run_tests()
  File "/data/localhome/vwkoppejan/software/python/anaconda2/envs/fipy2-env/lib/python2.7/site-packages/fipy/tests/testClass.py", line 236, in run_tests
    testLoader = loader_class()
  File "/data/localhome/vwkoppejan/software/python/anaconda2/envs/fipy2-env/lib/python2.7/unittest/main.py", line 94, in __init__
    self.parseArgs(argv)
  File "/data/localhome/vwkoppejan/software/python/anaconda2/envs/fipy2-env/lib/python2.7/unittest/main.py", line 149, in parseArgs
    self.createTests()
  File "/data/localhome/vwkoppejan/software/python/anaconda2/envs/fipy2-env/lib/python2.7/unittest/main.py", line 158, in createTests
    self.module)
  File "/data/localhome/vwkoppejan/software/python/anaconda2/envs/fipy2-env/lib/python2.7/unittest/loader.py", line 130, in loadTestsFromNames
    suites = [self.loadTestsFromName(name, module) for name in names]
  File "/data/localhome/vwkoppejan/software/python/anaconda2/envs/fipy2-env/lib/python2.7/unittest/loader.py", line 100, in loadTestsFromName
    parent, obj = obj, getattr(obj, part)
AttributeError: 'module' object has no attribute 'testFiPy'

**Comment by wd15** [comment]
@vwkoppejan, I don't know what is going on with the above traceback, sorry, but maybe try running the tests from a working version if FiPy using `python setup.py test` instead.

**Comment by vwkoppejan** [comment]
@wd15, I installed fipy using the conda recipe on the website. I've now cloned the fipy repo (develop branch) and from within the fipy env I've ran `python setup.py test`

I've added the log file in the attachment

[log-test.txt](https://github.com/usnistgov/fipy/files/1225319/log-test.txt)

I get 4 fails. The levelset ones I don't mind since I won't be using it. I'm not sure what's happening with the other ones.
